fhttp/http_test.go

```diff
@@ -493,11 +493,11 @@ func TestGenerateSize(t *testing.T) {
 // Many of the earlier http tests are through httprunner but new tests should go here
 
 func TestEchoBack(t *testing.T) {
-	p, m := DynamicHTTPServer(false)
+	m, a := DynamicHTTPServer(false)
```

Comment:
```
-      m, a := DynamicHTTPServer(false)
+	mux, addr := DynamicHTTPServer(false)
```
here and elsewhere

Reply:
I thought go was all about short names ? (those are test files)

Reply:
not fixing is fine